src/mutex.rs

```diff
@@ -7,7 +7,7 @@
 ///
 /// If there is no feature available for your Mutex type, you have to write one yourself. It should
 /// look something like this (for [`cortex_m`] as an example):
-/// ```
+/// ```ignore
```

Comment:
I also had to disable this doc-test, because I couldn't work out how to run tests with it enabled, perhaps I am missing something, or it could be feature gated with `cortex_m`?

Reply:
Hmm, it works fine for me ... Have you enabled the `cortexm` feature while running the tests?

```
cargo test --features 'std cortexm'
```